packages/core/test/convert.spec.ts

```diff
@@ -111,8 +110,8 @@ describe('Convert EthersJS Objects', () => {
     )
   })
 
-  describe('convertEthersTransactionReceipt', () => {
-    it('succeeds on anvil', async () => {
+  describe('succeeds on anvil', () => {
+    it('convertEthersTransactionReceipt', async () => {
```

Comment:
I switched the structure of the tests in this file slightly to make it easier to skip the live network tests in CI. The tests themselves remain unchanged